src/demo/DemoTestSearchService.java

```diff
@@ -294,6 +294,31 @@ private static void testSearchEntitiesByFilter() throws MambuApiException {
 			System.out.println("Warning: Cannot test filter by custom field. Client " + demoClient.getFullNameWithId()
 					+ " has no assigned custom fields");
 		}
+		
+		// Test GET Clients by Custom Field value using EQUALS_CASE_SENSITIVE
+		if (customFields != null && customFields.size() > 0) {
```

Comment:
The code for this scenario looks very similar to the code for the previous one and not only. Any reasons for not extracting parts of it to common places? Will it affect the readability for our SDK end-users in a negative way?

Also, can use !customFields.isEmpty() instead of customFields.size() > 0 or event better, we can have a utility class such as CollectionUtils and to implement a method like:


	public static boolean isEmpty(Collection<?> collection) {

		return collection == null || collection.isEmpty();
	}